tbtadm/controller.cpp

```diff
@@ -480,15 +482,17 @@ void tbtadm::Controller::createTree(ControllerInTree& controller,
             desc.emplace_back("Route-string: " + routeString + "\n");
             desc.emplace_back("Authorized: " + authorized(p) + "\n");
             desc.emplace_back("In ACL: " + inACL(p) + "\n");
-            desc.emplace_back("UUID: " + read(p / uniqueIDFilename));
+            desc.emplace_back("UUID: " + readAndTrim(p / uniqueIDFilename)
+                              + "\n");
```

Comment:
What's the point of this change?

Reply:
Oh, I see the commit description. OK, if this is what you prefer.

Reply:
Yes, otherwise we may have different output in there is no newline in the sysfs entry

Reply:
There is a new line if you eep it using `read()` instead `readAndTrim()`, yes?

Reply:
Yes, but for simulated environment there is no newline, so this really depends on sysfs entry. We should also verify that entry is more-less correct checking size, etc.

Reply:
So the simulation is bad simulation... sysfs must include a newline, as much as I know.